src/allmydata/scripts/create_node.py

```diff
@@ -344,6 +348,74 @@ def create_node(config):
         os.mkdir(basedir)
     write_tac(basedir, "client")
 
+    # if we're doing magic-wormhole stuff, do it now
+    if config['join'] is not None:
+        print >>out, "Opening wormhole with code '{}'".format(config['join'])
+        relay_url = config.parent['wormhole-server']
+        print >>out, "Connecting to '{}'".format(relay_url)
+
+        wh = wormhole.create(
+            appid=u"tahoe-lafs.org/lafs",
```

Comment:
This should probably be changed to `tahoe-lafs.org/tahoe-lafs/v1` (so as to be compatible with [this](https://github.com/LeastAuthority/leastauthority.com/pull/568/commits/aeed6f45198aee6e0647d11f3b185845a0c8a378#diff-0893f54dd902f12cade21bc3698a4019R341) and others).

On a related note, we might want to consider now whether this should be bumped up to "v2", or even be made more specific (perhaps so as to distinguish the "grid invites" of now from the "magic-folder invites" of the future -- maybe something like `tahoe-lafs.org/tahoe-lafs/invite` vs. `tahoe-lafs.org/tahoe-lafs/magic-folder/invite`?).